### PR TITLE
zizmor 0.1.1 (new formula)

### PR DESCRIPTION
**Pull request by miketheman**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

~~For the audit - there's something weird about [crates.io not allowing HTTP requests with no `'Accept: text/html'` header](https://github.com/rust-lang/crates.io/issues/788) - even with `curl` and other tools, so I was unable to complete the audit.~~

### CODE REVIEW

File: Formula/z/zizmor.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Zizmor < Formula
+  desc "CLI tool for finding security issues in GitHub Actions setups"
+  homepage "https://crates.io/crates/zizmor"
+  url "https://github.com/woodruffw/zizmor/archive/refs/tags/v0.1.1.tar.gz"
+  sha256 "d743533d553538b5392ea3bfc45b27d1b55606280babd9418f9cfe7af0273ec0"
+  license "MIT"
+
+  depends_on "rust" => :build
```
Comment (carlocab) [inline Formula/z/zizmor.rb:8]: This should fix your Linux failure:
```suggestion
  depends_on "pkg-config" => :build
  depends_on "rust" => :build

  on_linux do
    depends_on "openssl@3"
  end
```

Comment (miketheman) [reply Formula/z/zizmor.rb:9]: Thanks!

File: Formula/z/zizmor.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+class Zizmor < Formula
+  desc "CLI tool for finding security issues in GitHub Actions setups"
+  homepage "https://crates.io/crates/zizmor"
```
Comment (carlocab) [inline Formula/z/zizmor.rb:3]: We should probably just use the GitHub page as the `homepage` to avoid `brew audit` failures.

Comment (miketheman) [inline Formula/z/zizmor.rb:3]:
```suggestion
  homepage "https://github.com/woodruffw/zizmor"
```

**Review by carlocab — APPROVED** [review]
Thanks! Squashed your commits together for your. For future pull requests, please follow these guidelines from the [style guide](https://docs.brew.sh/Formula-Cookbook#commit):

> - One formula per commit; one commit per formula.
> - Keep merge commits out of the pull request.

**Comment by miketheman** [comment]
> Thanks! Squashed your commits together for your. For future pull requests, please follow these guidelines from the [style guide](https://docs.brew.sh/Formula-Cookbook#commit):

Thanks, will do! I was editing in the GH UI, and saw that the `autosquash` label was added automatically at one point, so thought things would "work out".

**Comment by p-linnane** [comment]
Thanks for the PR and stopping by @miketheman! Hope you're doing well.

**Comment by miketheman** [comment]
Thanks @p-linnane ! Doing well, thanks.

**Pull request merged**